systemservices/marketplace/src/types/version.ts

```diff
@@ -1,9 +1,6 @@
-import { Manifest } from "./manifest";
-
 export interface Version {
   versionHash: string;
   manifest: string;
   manifestProtocol: string;
-  manifestData: Manifest|null;
```

Comment:
how about changing this to `manifestData?: Manifest;` and after this change we can add the _manifestData_ directly into https://github.com/mesg-foundation/core/pull/896/files#diff-1497f2b1650c529c8f58159491cfe107R8 instead of recreating the object

Reply:
I feel it's safer to have multiple type. Like this, `manifestData` doesn't exist in Version and the developer cannot confuse between a not loaded `manifestData` and an not existing one (when the file is no more accessible from IPFS).